presto-record-decoder/src/main/java/io/prestosql/decoder/avro/AvroRowDecoderFactory.java

```diff
@@ -13,25 +13,36 @@
  */
 package io.prestosql.decoder.avro;
 
+import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
```

Comment:
I would prefer not to use kafka specific code in generic `presto-record-decoder`. I would prefer to introduce some extensions points to `AvroRowDecoderFactory` and use them in `presto-kafka` module to use kafka schema registry there.